src/env.h

```diff
@@ -1025,6 +1025,14 @@ class Environment : public MemoryRetainer {
   void Exit(int code);
   void ExitEnv();
 
+  static bool AllowWasmCodeGenerationCallback(v8::Local<v8::Context> context,
+                                              v8::Local<v8::String>);
+  static bool ShouldAbortOnUncaughtException(v8::Isolate* isolate);
+  static v8::MaybeLocal<v8::Value>
```

Comment:
i am surprised that this passes our lint

Reply:
i was too lol